frontend/packages/dev-console/src/components/helm/HelmReleaseRow.tsx

```diff
@@ -34,9 +43,15 @@ const HelmReleaseRow: React.FC<HelmReleaseRowProps> = ({ obj, index, key, style
         <Status status={_.capitalize(obj.info.status)} />
       </TableData>
       <TableData className={tableColumnClasses.chartName}>{obj.chart.metadata.name}</TableData>
+      <TableData className={tableColumnClasses.chartVersion}>
+        {obj.chart.metadata.version}
+      </TableData>
       <TableData className={tableColumnClasses.appVersion}>
         {obj.chart.metadata.appVersion}
       </TableData>
+      <TableData className={tableColumnClasses.kebab}>
+        <Kebab options={menuActions} key={`${obj.name}-${obj.version}`} />
```

Comment:
This PR adds the upgrade action to just the details page. But we also have helm release details page and topology where we show helm release and actions need to be added there as well.

Reply:
I know the story AC doesn't explicitly mentions this but we did discuss this during the grooming of the stories and we updated the AC of `delete` story. Maybe we missed updating the AC for this one.